ros/api/v1/call_to_action.py

```diff
@@ -1,15 +1,15 @@
 from flask_restful import Resource
 from flask import request
 from ros.lib.utils import (
-    identity, system_ids_by_account)
+    identity, system_ids_by_org_id)
 from ros.lib.models import (
     PerformanceProfile, db)
 
 
 class CallToActionApi(Resource):
     def get(self):
         account_number = identity(request)['identity']['account_number']
-        system_query = system_ids_by_account(account_number).filter(PerformanceProfile.number_of_recommendations > 0)
+        system_query = system_ids_by_org_id(account_number).filter(PerformanceProfile.number_of_recommendations > 0)
```

Comment:
```suggestion
        system_query = system_ids_by_org_id(org_id).filter(PerformanceProfile.number_of_recommendations > 0)
```

Method  `system_ids_by_org_id` accepts `org_id` whereas you passed `account_number`. 
Similarly here also you need to fetch `org_id` value from identity header?

Reply:
Oh I missed this 🤦🏻‍♀️.